src/org/zalando/stups/mint/worker/external/http.clj

```diff
@@ -0,0 +1,21 @@
+(ns org.zalando.stups.mint.worker.external.http
+  (:require [clj-http.client :as client]))
+
+(def ^:private default-http-options
+  {:socket-timeout 10000
```

Comment:
Is 10s enough for every use case in mint-worker? We could increase it to 30s or even 60s I would say to catch any latency spikes or slow network. The whole mint worker is just asynchronous job, so it's better to complete in 30s successfully than failing early after 10s (as no "customer" is waiting). The goal is just to have "some reasonable" timeouts instead of none.